### ISSUE TITLE
Glob pattern doesn't work

### ISSUE THREAD

**Issue by exophunk** [anteriovieira/nuxt-sass-resources-loader#14]
### Overview of the problem

- **Nodejs** version: [9.5.0]
- **Nuxtjs** version: [1.4.0]
- **nuxt-sass-resources-loader** version: [2.0.0]

### Description

No matter what I try, I cannot get glob patterns (`**/*`) to work.
With a single file include (no **/*), it all works, so paths are fine.

My nuxt.config.js :

```
module.exports = {

    srcDir: 'client/',

    modules: [
        ['nuxt-sass-resources-loader', {
            resources: [
                '@/assets/scss/mixins/**/*.scss',
            ],
        }],
    ],
};

```

### Expected behavior

Import should work.

### Actual behavior

Error:

```
 ERROR  Nuxt error

  Error: Cannot resolve "@/assets/scss/mixins/**/*.scss" from "/Users/...../project/client/assets/scss/mixins/**/*.scss"
```

**Comment by tomexx** [comment]
same here

**Comment by anteriovieira** [comment]
Sorry for the delay, this was fixed in `v2.0.1`

Thank you for reporting the problem.
